zerver/lib/actions.py

```diff
@@ -2636,9 +2636,26 @@ def update_flag(um, should_set, flag):
     for um in changed_ums:
         um.save(update_fields=['flags'])
 
+def update_to_dict_cache(changed_messages):
+    # type: (List[Message]) -> List[int]
+
+    # Update the message as stored in the to_dict
+    # cache (for shunting the message over to Tornado in the old
+    # get_messages API).
```

Comment:
this comment still isn't right; the `to_dict` cache is used for serving messages :)